sdk/core/core-auth/src/tracing.ts

```diff
@@ -4,38 +4,57 @@
 // The interfaces in this file should be kept in sync with those
 // found in the `@azure/core-tracing` package.
 
+// NOTE: these were taken directly from the @opentelemetry/api package as of v0.16
+
+/**
+ * Attributes for a span.
+ */
+export interface Attributes {
+  [attributeKey: string]: AttributeValue | undefined;
+}
+/**
+ * Attribute values may be any non-nullish primitive value except an object.
+ *
+ * null or undefined attribute values are invalid and will result in undefined behavior.
+ */
+export declare type AttributeValue =
+  | string
+  | number
+  | boolean
+  | Array<null | undefined | string>
+  | Array<null | undefined | number>
+  | Array<null | undefined | boolean>;
+
 /**
  * An interface that enables manual propagation of Spans.
  */
 export interface SpanOptions {
-    /**
-     * The SpanContext that refers to a parent span, if any.
-     * A null value indicates that this should be a new root span,
-     * rather than potentially detecting a span via a context manager.
-     */
-    parent?: SpanContext | null;
-    /**
-     * Attributes to set on the Span
-     */
-    attributes?: {
-        [key: string]: unknown;
-    };
+  /**
+   * The SpanContext that refers to a parent span, if any.
+   * A null value indicates that this should be a new root span,
+   * rather than potentially detecting a span via a context manager.
+   */
+  parent?: SpanContext | null;
```

Comment:
since `SpanOptions` in OT doesn't have `parent` anymore, this isn't structurally compatible

Reply:
Yes, will remove (especially given what we discussed with just passing `Context` instead)

Reply:
Do you have to wait to remove it because of compile errors until you update all dependents?

Reply:
I know this is an old comment, but yes. Although the big prep PR that went in should finally make this easy. :)